SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialEventTracer.cpp

```diff
@@ -115,14 +149,41 @@ SpatialEventTracer::SpatialEventTracer(const FString& WorkerId)
 	}
 
 	FolderPath = EventTracePath;
-	const FString FullFileName = FString::Printf(TEXT("EventTrace_%s_%s.trace"), *WorkerId, *FDateTime::Now().ToString());
-	const FString FilePath = FPaths::Combine(FolderPath, FullFileName);
-
 	IPlatformFile& PlatformFile = FPlatformFileManager::Get().GetPlatformFile();
+	const FString FileName = TEXT("event-log");
+	const FString FileExt = TEXT(".etlog");
```

Comment:
Nice! @Mudit-G could you change the (default?) suffix of event logs to this in squid too?